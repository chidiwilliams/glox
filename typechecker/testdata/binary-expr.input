var x = 10;
var y = 20;
var z: string = x * 10 + y;
z;
