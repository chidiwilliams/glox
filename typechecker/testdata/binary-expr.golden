type error on line 3: expected 'string' type, but got 'number'
